Config.lua

```diff
@@ -19,12 +19,12 @@ local soundpacks = {
 }
 
 local zones = {
-    ["arena"] = ARENA,
-    ["pvp"] = BATTLEGROUNDS,
-    ["party"] = DUNGEONS,
-    ["raid"] = RAIDS,
-    ["scenario"] = SCENARIOS,
-    ["none"] = BUG_CATEGORY2
+    ["party"] = { name = DUNGEONS, order = 1 },
+    ["raid"] = { name = RAIDS, order = 2 },
```

Comment:
I would like Raid to go before Dungeons as that is what the addon is mainly intended for. The remaining order looks good.